core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/kubernetesModels.scala

```diff
@@ -45,44 +43,81 @@ case class NetworkFields(networkName: NetworkName,
  *  see: https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#Cluster.Status
  */
 
-//TODO: possibly consolidate Kubernetes status when monitoring is implemented
```

Comment:
I finally understand what you meant by the comment above (better late than never, eh 😛). I agree with it and would vote for keeping it (perhaps with "Kubernetes status" replaced by "KubernetesClusterStatus'es and NodepoolStatus'es").